.changeset/lemon-gifts-crash.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-scaffolder-common': patch
+'@backstage/plugin-scaffolder-react': patch
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
then this one should go away if we don't make any changes to `scaffolder-backend`

```suggestion
```